bin/release.py

```diff
@@ -296,8 +296,14 @@ def prepare_release_artifact(build_job_number, work_dir):
         raise Exception(
             f"BASE_IMAGE not found in nginx-version-info: {nginx_version_info}"
         )
+    if 'ARCH' not in variables:
```

Comment:
Please add a comment, either just above this `if` or as an additional paragraph after the comment above, saying what `ARCH` is, how it is used, and which values it can have.